### PR TITLE
Add new clang with builtin gcc 8.3.1 std lib

### PR DESCRIPTION
**Pull request by white238**
host-configs coming soon

### CODE REVIEW

**Comment by white238** [comment]
Discovered axom isnt updating in the spack mirror so it has been stuck since may.  Investigating work around.

File: scripts/uberenv/packages/serac/package.py [hunk 1]
```diff
@@ -394,8 +394,9 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
                 cfg.write("# Root directory for generated developer tools\n")
                 cfg.write(cmake_cache_entry("DEVTOOLS_ROOT",devtools_root))
 	
-        if socket.gethostname().startswith("lassen"):
-            cfg.write("# Docs dont work on lassen, there is a prompt that waits for user input\n")
+        hostname = socket.gethostname()
+        if hostname.startswith("lassen") or hostname.startswith("rzansel"):
```
Comment (joshessman-llnl) [inline scripts/uberenv/packages/serac/package.py]: Should we have Sierra here so all the CORAL machines are covered?

**Comment by white238** [comment]
LGTM

**Review by joshessman-llnl — APPROVED** [review]

**Review by jamiebramwell — APPROVED** [review]

**Comment by white238** [comment]
LGTM

**Pull request merged**